src/SonarScanner.MSBuild.PreProcessor/ProcessedArgs.cs

```diff
@@ -118,6 +84,44 @@ public string PropertiesFileName
             }
         }
 
+        protected /* for testing */ ProcessedArgs() { }
```

Comment:
I would remove the comment.

Reply:
This is a common pattern in S4NET. I would keep it for consistency.

Reply:
Ok, on my side I would remove it consistently :).

The problem I see is that it often causes confusion. The comment doesn't tell if the method is only used for testing or if the access modifier is different for testing.